authors/apps/authentication/views.py

```diff
@@ -25,6 +28,36 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
         serializer.save()
+
+        # These are the values about the user that we need to send an email
+        subject = "Confirm your account"
+        template_name = 'verify_email.html'
+
+        # Prepare to generate a token to be used in verifying that the user
+        # `call_back_url` is the url that should be redirected to after
+        # verifying the user.
+        import os
```

Comment:
@ssewilliam, y are you importing these here? I think importing them at the beginning of the file would be much cleaner and reusable